cc.arduino.arduinoide.json

```diff
@@ -0,0 +1,236 @@
+{
+    "app-id": "cc.arduino.arduinoide",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.openjdk10"
+    ],
+    "rename-icon": "arduino",
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "finish-args": [
+        "--socket=x11",
```

Comment:
Does it actually work without --device=all?

Reply:
Good point! Yes it needs it.